src/pollux/Pollux.ts

```diff
@@ -275,7 +275,7 @@ export default class Pollux implements IPollux {
         }
         const statusListDecoded = this.extractEncodedList(revocation)
         const bitstring = new Bitstring({ buffer: statusListDecoded })
-        return bitstring.get(statusListIndex + 1)
+        return bitstring.get(statusListIndex)
```

Comment:
I'd expect a unit test to need changing to reflect this?

Reply:
There's unit tests actually for that but they were mocking the index value.
I've updated the test though, agree